ui/components/src/DateTimeRangePicker/TimeRangeSelector.tsx

```diff
@@ -19,6 +19,16 @@ import { formatAbsoluteRange } from './utils';
 
 const DATE_TIME_FORMAT = 'yyyy-MM-dd HH:mm:ss';
 
+function heightToPaddingY(height: string): string {
+  const heightNum = parseFloat(height);
+
+  if (heightNum >= 20) {
```

Comment:
Why `20` here? Is this number dependent on something that can change in a MUI theme?

Reply:
Feel free to ignore this if the alternative css suggestion removes the need for it.